powergenome/time_reduction.py

```diff
@@ -288,7 +288,7 @@ def kmeans_time_clustering(
     # dropped off from original data set  due to rounding
     RMSE = {
         col: np.linalg.norm(
-            np.sort(input_data.truncate(after=len(FullLengthOutputs))[col].values)
+            np.sort(input_data.truncate(after=len(FullLengthOutputs)-1)[col].values)
```

Comment:
@nspatank I get that the index point for truncation needs to be increased. But if the length of `FullLengthOutputs` is less than 8760, does it make sense to create a truncated version of `input_data` earlier in this function? 

At some point I really want to come back and clean up/rewrite this function to make sure it works correctly. Or even figure out a simple test data set that we can compare results with.